apps/admission-webhook-k8s/webhook-secret-example.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-tls
+  namespace: nsm-system
```

Comment:
Do we really need this file?
I don't think there is too much value in example for standard k8s TLS secret.

Reply:
Removed, found a way to create this secret using `kubectl`